packages/cli/test/cli/CLITest.ts

```diff
@@ -63,6 +63,22 @@ describe('CLI', function () {
     )
   })
 
+  it('respects globs', async function () {
+    const pathInFixtures = (dirPath: string): string =>
+      pathResolve(__dirname, '..', 'fixtures', 'glob-test', dirPath)
+    const { status, stdout, stderr } = await runCodemodCLI([
+      '--dry',
+      pathInFixtures('**/*.js'),
+      `!${pathInFixtures('omit.js')}`,
+    ])
+
+    strictEqual(status, 0)
+    strictEqual(stdout.includes('abc.js'), true)
```

Comment:
When these tests fail, the assertion doesn't provide great user feedback. Jest has a much richer set of matchers than `assert`, just fyi. :smile:

Reply:
Yep, I agree. I haven't been able to use jest on this package because it conflicts with `@codemod/cli`'s `require` extensions.